src/components/search-box/search-box.tsx

```diff
@@ -66,71 +70,97 @@ export const SearchBox = forwardRef<HTMLFormElement, SearchBoxProps>(
       changeHandler(event);
     };
 
+    const mergedRefs = useMergeRefs([forwardedRef, refs.setReference]);
+
     return (
       <>
         <Form.Root
+          {...getReferenceProps()}
           role="search"
           onSubmit={submitHandler}
           onReset={resetHandler}
-          ref={forwardedRef}
-          className={clsx('lui-search-box-form', classNames?.form)}
+          ref={mergedRefs}
+          className={clsx('lui-search-box', classNames?.form, open && 'lui-suggestions-open')}
           {...elementProps}
         >
-          <Form.Field name={fieldName}>
-            {labels?.label && (
-              <Form.Label className={clsx('lui-search-box-label', classNames?.label)}>
-                {labels.label}
-              </Form.Label>
-            )}
-
-            <Form.Control asChild>
-              <input
-                autoComplete="off"
-                aria-autocomplete="none"
-                ref={refs.setReference}
-                onChange={inputChange}
-                onFocus={() => {
-                  setOpen(!!inputValue);
-                }}
-                {...getReferenceProps()}
-                value={inputValue}
-                onKeyDown={handleKeyDown}
-                className={clsx('lui-search-box-input', classNames?.input)}
-                placeholder={labels?.placeholder}
-              />
-            </Form.Control>
-          </Form.Field>
-
           <Form.Submit asChild>
             <button
               type="submit"
-              className={clsx('lui-search-box-submit', classNames?.submit)}
+              className={clsx('lui-search-box-submit lui-search-box-button', classNames?.submit)}
               ref={submitRef}
             >
-              {submitIcon && (
+              {labels?.submit && <span className="lui-sr-only">{labels.submit}</span>}
```

Comment:
if you add this as an `aria-label` attribute to meet the accessibility, you won't need the `lui-sr-only` styles

Reply:
Good suggestion. Done